automation/taskcluster/decisionlib.py

```diff
@@ -21,6 +21,8 @@
 import re
 import subprocess
 import sys
+from typing import List
```

Comment:
Out of curiosity, any reason why you opted for typing here instead of a classic list?

Reply:
Yeah! I don't actually use `List` to create a list, I'm just using in `DockerWorkerTask` to denote a property that will be a list. I set that up so that my IDE would provide some auto-completion while I worked on this feature, and give me a hint if I wired up my list wrong

Reply:
Sounds good. I'm okay with keeping it should you want to, was just curious for the reasoning.